### DISCUSSION TITLE
Why doesn't Biome show up in my default formatting options in vscode?

### DISCUSSION THREAD

**Discussion by painotpi** [biomejs/biome#1618] [Q&A] [answered]
I've installed the biome extension, but when I select "Format Document With", I don't see Biome in the list,

![image](https://github.com/biomejs/biome/assets/3349253/08d05209-da17-489a-902f-393aba3cfbeb)

Details:

OS: Windows 11 build 22621.3007
Editor: VSCode v1.84.2

**Comment by ematipico** [comment]
It's very difficult to help you with just a screenshot. Can you give us more info? Did you install Biome in your project? What version? Did you create a confirmation file?

**Reply by painotpi** [reply]
Yeah, seems like it was what @canadaduane hinted at. It just showed up a week back. I wasn't signed it to my personal account so never got around to replying here.

**Comment by canadaduane** [comment, marked as answer]
I've noticed that VSCode sometimes "forgets" that Biome is available as a formatter. I'm not sure how it loses track, or why it sometimes seems to "fix itself" after a while. I will try to pay more attention to this phenomenon.

**Reply by painotpi** [reply]
Yeah, seems like this was it. It just showed up a week back. I wasn't signed it to my personal account so never got around to replying here.

**Comment by elieven** [comment]
This is still happening
